cmd/tikv-ctl/Cargo.toml

```
@@ -70,7 +70,7 @@ pd_client = { path = "../../components/pd_client", default-features = false }
 prometheus = { version = "0.13", features = ["nightly"] }
 protobuf = { version = "2.8", features = ["bytes"] }
 raft = { version = "0.7.0", default-features = false, features = ["protobuf-codec"] }
-raft-engine-ctl = { git = "https://github.com/tikv/raft-engine.git" }
+raft-engine-ctl = { git = "https://github.com/tikv/raft-engine.git", branch = "master" }
```

BTW this is unnecessary. It will automatically use the default branch of the repo.
```suggestion
raft-engine-ctl = { git = "https://github.com/tikv/raft-engine.git" }
```